app/src/main/java/com/iamandiiana/newsreader/feature/newslist/MainActivity.java

```diff
@@ -1,10 +1,11 @@
-package com.iamandiiana.newsreader;
-
-import androidx.appcompat.app.AppCompatActivity;
+package com.iamandiiana.newsreader.feature.newslist;
 
 import android.os.Bundle;
 
-import com.iamandiiana.newsreader.ui.main.MainFragment;
+import androidx.appcompat.app.AppCompatActivity;
+
+import com.iamandiiana.newsreader.R;
+import com.iamandiiana.newsreader.feature.newslist.fragment.NewsListFragment;
 
 public class MainActivity extends AppCompatActivity {
```

Comment:
Let's also rename this activity to something more specific

Reply:
Done